src/services/validator.service.ts

```diff
@@ -0,0 +1,58 @@
+import config from '~app/common/config';
+import { getRequest } from '~root/services/httpApi.service';
+
```

Comment:
In operator we had the following to handle pagination:

type OperatorsListQuery = {
  page?: number,
  search?: string,
  type?: string[],
  perPage?: number
  ordering?: string,
  dkgEnabled?: boolean,
};

type OperatorValidatorListQuery = {
  page?: number,
  perPage?: number
  operatorId: number,
};


Possibly we need something like that here.

Reply:
page and other params are managed in MyAccount.store and are not passed as param for function call